e2e-fixtures/hello-world-ingress/EXAMPLE-config-pt2.yaml

```diff
@@ -0,0 +1,5 @@
+# 1. Copy this file to ./config.yaml
+# 2. Set you unique FQDN in the "value" field below
+- op: replace
+  path: /spec/rules/0/host
+  value: <UNIQUE SUBDOMAIN>.ngrok.io
```

Comment:
if this is just to override those bezek specific ones, feel free to rip all that apart and change up the base file. These got out of date near the end since I was the only one using them and all the modulset stuff changed really quickly